src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenuTriggerButton/index.tsx

```diff
@@ -14,13 +15,17 @@ export const GroupCardDropdownMenuTriggerButton: FC<IGroupCardMenuButtonProps> =
   handleRenameClick,
 }) => {
   return (
-    <DropdownMenu>
-      <DropdownMenuTrigger asChild>
-        <Button variant="ghost" className="w-12">
-          <Icon icon={MenuKebabIcon} size={4.5} />
-        </Button>
-      </DropdownMenuTrigger>
-      <GroupCardDropdownMenuContent handleRenameClick={handleRenameClick} />
-    </DropdownMenu>
+    // Wrapping dialog menu with DialogRoot. Trigger is in GroupCarDropdownMenuContent
```

Comment:
I'm sorry, but could you please clarify this comment? It's a bit difficult to understand what it's trying to say.

I may be like:
> The `<DropdownMenu>` must be wrapped with the `<DialogRoot>` to prevent the dialog from closing unintentionally when the dropdown menu is closed. The trigger for the dialog can be found in the `<GroupCarDropdownMenuContent>`.